src/main/webapp/user-page.html

```diff
@@ -40,7 +40,14 @@ <h1 id="page-title">User Page</h1>
     </form>
     <hr/>
 
-    <div id="message-container">Loading...</div>
```

Comment:
At a quick glance, it's not clear why you're renaming this <div>. Is the message container being removed from the user page? Is that intended?

I see that fetchMessages() is still being called in javascript, which seems to imply that we still want to populate the page with messages; did you intend to remove that too?

Reply:
Hi Matt! I'm not sure, but in that step it just told me to copy it and not worry about it since it would work in conjunction with Mario's message servlet (which I'm assuming deals with renaming).

Reply:
The procedure says to add the `<div id="about-me-container">`, but doesn't say to remove the existing `<div>`, unless I'm missing something.